content/sensu-core/2.0/guides/influx-db-metric-handler.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "How to populate InfluxDB metrics using handlers"
+linkTitle: "Populating InfluxDB with Handlers"
+weight: 125
+version: "2.0"
+product: "Sensu Core"
+platformContent: false
+menu:
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What are Sensu handlers?
+
+Sensu event handlers are actions executed by the Sensu server on [events][1].
+
+## Why use a handler?
+
+Handlers can be used for sending email alerts, creating or resolving an incident
+(e.g., in PagerDuty, ServiceNow, etc), or storing metrics in a time-series
+database (e.g., Graphite, InfluxDB).
+
+## Using a handler to populate InfluxDB
+
+The purpose of this guide is to help you populate Sensu metrics into the time
+series database [InfluxDB][2]. Metrics can be collected from check output or
+from the Sensu StatsD Server. For more information on how to send StatsD metrics
+to Sensu, check out [this guide][3].
+
+### Installing the handler command
+
+The first thing you'll want to do is create an executable script named
+`handler-influx-db`, which will be responsible for populating a configurable
+InfluxDB with Sensu metrics. The source code of this script is available on
+[GitHub][4] and can easily be compiled or [cross compiled][5] using the
+[Go tools][6]. The generated binary will be placed into one of the Sensu backend
+[`$PATH` directories][7], more precisely `/usr/local/bin`.
+
+{{< highlight shell >}}
+# From the local path of the sensu-go repository
+go build -o /usr/local/bin/handler-influx-db handlers/influx-db/main.go
+{{< /highlight >}}
+
+### Creating the handler
+
+Now that our handler command is installed, the second step is to create a
+handler that we will call `influx-db`, which is a **pipe** handler that pipes
+event data into our previous script named `handler-influx-db`. We will also pass
+the database name, address, username, and password of the InfluxDB you wish to
+populate.
+
+{{< highlight shell >}}
+sensuctl handler create influx-db \
+--type pipe \
+--command 'handler-influx-db' \
+  --addr '123.4.5.6' \
+	--db-name 'myDB' \
```

Comment:
Is the indentation on purpose? Looks a bit strange but maybe it's just me!

Reply:
Nope, totally accidental 😅